python/ray/tests/test_utils.py

```diff
@@ -38,9 +40,43 @@ def test_get_or_create_event_loop_new_event_loop():
         assert loop is not None, "new event loop should be created."
 
 
+def test_try_import_each_module():
```

Comment:
can you write an e2e tests?

Reply:
maybe one with successful, and the other with failed version.

Reply:
Sure -- can I defer to when we're more confident this meets user needs? Wanting to get feedback, then properly productionize.

Seems low risk since it's gated by env var, LMK thoughts

Reply:
I think e2e test is the mininal we need to merge the PR. and it would take 10m to write it?

Reply:
```python3
def test_preload(start_ray_cluster):
    cluster = start_ray_cluster
    cluster.add_node(_system_config={"preload": blah})
    # create an actor and make sure modules are already imported
```

Reply:
ah didn't know that's what you meant by e2e test. sure i'll add

Reply:
yeah lol. some people call it unit test I guess..